### PR TITLE
Provider compat CI: remove testcase with corrupted RSA ciphertext

### PR DESCRIPTION
**Pull request by t8m**
This testcase does not work anymore after PR #13817 was merged.

### CODE REVIEW

**Comment by t8m** [comment]
https://github.com/t8m/openssl/actions/runs/3679031867/jobs/6222999981 - fix confirmed

**Review by beldmit — APPROVED** [review]
LGTM

File: .github/workflows/fips-provider.yml [hunk 1]
```diff
@@ -92,6 +92,9 @@ jobs:
       - name: copy the provider
         run: |
           cp -a build/providers/fips.so build-3.0/providers/fips.so
+      - name: removing incompatible testcase
```
Comment (paulidale) [inline .github/workflows/fips-provider.yml:95]: This is not a good way to omit a test.  Do it with a FIPSversion conditional in the test data.

**Review by paulidale — CHANGES_REQUESTED** [review]

File: .github/workflows/fips-provider.yml (hunk 1 above)
Comment (t8m) [reply .github/workflows/fips-provider.yml:95]: So should I do a PR against 3.0 branch?

Comment (t8m) [reply .github/workflows/fips-provider.yml:95]: Alternative PR (against 3.0 branch) in #19904

**Comment by paulidale** [comment]
Much prefer the alternative.  Closing this.
Reopen in case of disagreement.

**Pull request closed without merging**